### DISCUSSION TITLE
Installation done horribly wrong, resulting in weird Oh My Posh.

### DISCUSSION THREAD

**Discussion by tsoruu** [JanDeDobbeleer/oh-my-posh#4155] [Troubleshoot]
Steps: Install Oh My Posh from Microsoft Store, Install  Oh My Posh deprecated PowerShell module, uninstall Oh My Posh deprecated PowerShell Module, enable Oh My Posh installed from Microsoft Store
![Screenshot 2023-08-13 090811](https://github.com/JanDeDobbeleer/oh-my-posh/assets/83799574/98767e6f-3d69-49a0-a780-642dc774568c)
regarding a similar discussion, idk how to search for this problem... sry...
I'm using the Atomic Theme.

**Comment by JanDeDobbeleer** [comment]
@tsoruu all you need is a Nerd Font and configure your terminal to use it. https://ohmyposh.dev/docs/installation/fonts

**Reply by tsoruu** [reply]
ohh, okie dokie, thanks for the help!
